system/libarc32_arduino101/drivers/soc_i2s.c

```diff
@@ -228,7 +229,7 @@ static void i2s_dma_cb_block(void *num)
 DRIVER_API_RC soc_i2s_config(uint8_t channel, struct soc_i2s_cfg *cfg)
 {
 	uint32_t reg;
-	uint16_t sample_rate;
+	//	uint16_t sample_rate;
```

Comment:
If there is a good reason for these commented lines (232, 251), can you add more comments to explain when/why they might be needed?